numpy/core/tests/test_numpy_12905.py

```diff
@@ -0,0 +1,23 @@
+
+import sys
+import pickle
+
+import numpy as np
+from numpy.testing import (
+     run_module_suite, assert_, assert_equal, 
+)
+
+def test_numpy_12905():
```

Comment:
I think this should move to `numpy\core\tests\test_multiarray.py:TestPickling.test_datetime64_byteorder

Reply:
Done.